src/main/java/project_16x16/projectiles/MagicProjectile.java

```diff
@@ -109,8 +109,13 @@ public void hit(CollidableObject collision) {
 	private void setParticleAnimation(SideScroller a) {
```

Comment:
Not sure if this really does have to deal with the PR, but this method is the same to its counterpart in the MagicSourceObject class. Can this be DRY'd with an abstract class?